src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,33 @@
 package mate.academy;
 
+import mate.academy.DAO.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book newBook = new Book(null, "Clean Code", new BigDecimal("19.99"));
```

Comment:
You should not use 'null' for the id when creating a new `Book` object. The 'id' should be autogenerated by the database, and you should not set it manually when creating a new entity.